core/sql/cli/Statement.cpp

```diff
@@ -2102,6 +2102,30 @@ Statement * Statement::getCurrentOfCursorStatement(char * cursorName)
 
 }
 
+//////////////////////////////////////////////////////////////////
+////
+//// A helper function to show buffer in HEX
+////
+//// ///////////////////////////////////////////////////////////////
+
+static char *stringToHex(char * out, Int32 outLen, char * in, Int32 inLen)
```

Comment:
Could this function please be declared in some place like exp/ExpError.h (with an implementation in ExpError.cpp)? I see this function implemented twice in this PR and something similar in a previous PR. I think this is not advisable according to our coding guidelines.